tests/disas/winch/x64/epoch/func.wat

```diff
@@ -23,7 +23,7 @@
 ;;       cmpq    %rcx, %rdx
 ;;       jb      0x51
 ;;   44: movq    %r14, %rdi
-;;       callq   0x126
+;;       callq   0x127
```

Comment:
And here, we've somehow lost a byte again, but without any other changes to the file?